### DISCUSSION TITLE
Teleskoplader Arm lässt sich nicht ausfahren

### DISCUSSION THREAD

**Discussion by JimmyBeam1990** [Mogli12/KeyboardSteer#746] [Previous version]
Moin ihr lieben,

nach einer sehr langen suche nach modkonflikten bin ich auf VCA gekommen und habe es ausgetestet. Clean moderner und auf der original Karte geladen und ausgetestet. Es lässt sich bei einem Teleskoplader nicht mehr der arm ausfahren. 

Jemand eine Idee?? Inputbindings.xml habe ich schon aus dem Ordner entfernt, ergab aber keinen Erfolg.

**Comment by Mogli12** [comment]
Hallo,

Du hast wohl ein Problem mit deiner inputBinding.xml. Das steht hier auch bereits schon auf der Startseite, die aber niemand zu lesen scheint.

Benenne einfach dein inputBinding.xml mal um.

Viele Grüße

**Comment by JimmyBeam1990** [comment]
Inputbindings.xml habe ich schon aus dem Ordner entfernt, ergab aber keinen Erfolg.

**Comment by FarmerMike87** [comment]
> Inputbindings.xml habe ich schon aus dem Ordner entfernt, ergab aber keinen Erfolg.

Ich denke das es trotz allem nur ne Doppelbelegung ist, den der TeleArm funktioniert mit VCA ohne Probleme.
